valmlord/gulpify
# Feature/tasks separation

## Description
This pull request includes two main features:
1. Division of tasks into separate files for better organization and maintainability.
2. Addition of a base SCSS compile feature and implementation of a task to load JSON data.

## Changes
### Commit 1: Divided the task into separate files
- Refactored `gulpfile.js` to split tasks into individual files within a `tasks` directory.
- Updated the main `gulpfile.js` to require and use these separate task files.

### Commit 2: Add base SCSS compile feature and implement JSON data load task
- Added `gulp-sass` packages to `package.json`.
- Created a new task file for compiling SCSS to CSS.
- Created a new task file for loading and processing JSON data.

## How to Test
1. Execute the `gulp` command to ensure all tasks are loaded correctly.
2. Test the SCSS compile task by placing SCSS files in the appropriate directory and running the task.
3. Test the JSON data load task by placing a JSON file in the appropriate directory and running the task.

## Additional Notes
- These updates aim to improve the project structure and add new features for SCSS compilation and JSON data handling.
